raster_api/infrastructure/config.py

```diff
@@ -54,7 +54,7 @@ class vedaRasterSettings(BaseSettings):
         description="Name or ARN of the AWS Secret containing database connection parameters",
     )
 
-    raster_data_access_role_arn: Optional[str] = Field(
+    data_access_role_arn: Optional[str] = Field(
```

Comment:
the `RASTER` part of the `VEDA_RASTER_` environment prefix was recently stripped (https://github.com/NASA-IMPACT/veda-backend/blob/develop/raster_api/infrastructure/config.py#L86). We may want to bring that back but for now we need that `raster` in the config field name.

Reply:
I'm not sure I follow - this change is just so that both the ingest and raster APIs can access the same environment variable. Is there another reason we need to keep `raster_` here?

Reply:
The current env configuration in the secrets manager for our deployed instances is `VEDA_RASTER_DATA_ACCESS_ROLE_ARN`. We can modify that if needed but we'd also need to make sure we manage that change for all our instances including ghgc. 

Might be simpler to have the ingest assume a raster data access role (which is technically the reason it is used in ingest anyway).